src/stores/A_store.js

```diff
@@ -153,6 +153,45 @@ class A_store extends ReduceStore {
                 }
             }
 
+            case ActionTypes.DSTARLITE: {
+                const {player, end, pathGrid} = state;
+                const {grid} = pathGrid;
```

Comment:
i'd say you should just pull this during the destructure in the line above, since you aren't using `pathGrid` for anything else in here, *but*: